runhouse/main.py

```diff
@@ -129,14 +130,54 @@ def _start_server(
     )
 
     try:
+
         # Open and read the lines of the server logfile so we only print the most recent lines after starting
         f = None
         if screen and Path(Cluster.SERVER_LOGFILE).exists():
             f = open(Cluster.SERVER_LOGFILE, "r")
             f.readlines()  # Discard these, they're from the previous times the server was started
 
         # We do these one by one so it's more obvious where the error is if there is one
-        for cmd in cmds:
+        for i, cmd in enumerate(cmds):
+
+            last_cmd = i == len(cmds) - 1
+            if last_cmd:
```

Comment:
Iterating over all the commands to pull out the one that you want to manipulate is a bit roundabout. Could you just run the `command -v screen` before `Cluster._start_server_cmds` and then make the `screen` argument to `Cluster._start_server_cmds` accept either `None`, `screen`, or `nohup` to construct the correct command there?